src/main/java/org/cyclops/integrateddynamics/part/aspect/write/AspectWriteBase.java

```diff
@@ -61,6 +66,14 @@ public AspectWriteBase(ModBase mod, ModBase modGui, String unlocalizedTypeSuffix
             }
         } else if(!writerState.isDeactivated()) {
             onDeactivate(partTypeWriter, target, writerState);
+            postVariableContentsUpdatedEvent(target.getCenter());
+        }
+    }
+
+    private static void postVariableContentsUpdatedEvent(PartPos partPos){
```

Comment:
Is this event call really needed again here? Are there cases where the call from `PartTypeWriteBase` is not sufficient? If so, do you think it would be possible to combine them somehow?

Reply:
No, I don't think it is necessary -- I think perhaps I thought it was at some point because I had a network that was in a weird state due to some other stuff I'd been doing.
I've removed it.